Third-party-libraries.md

```diff
@@ -212,7 +215,43 @@ Note that we don't have an automatic way to detect outdated libraries in `depend
 
 ## Add dependencies
 
-Note that all dependencies must be compatible with our Apache 2 license. All additions must also be approved by @vojtechjelinek.
+### Requirements for new dependencies
+
+#### Checklists
+
+All new dependencies must be approved by @oppia/dependency-reviewers, who will check for the following criteria. Each new dependency must satisfy all of the conditions on one of the following lists:
+
+**For Large Open-Source Dependencies:** Open-source software is often maintained by small groups of people and open to contributions from pretty much anyone. Therefore, we evaluate its safety by its security practices and history. Reviewers may use [OpenSSF Scorecards](https://scorecard.dev) to automate some of these checks.
+
+* [ ] **Actively maintained based on recent commit history:** (do NOT rely on OpenSSF scorecard, which counts automated commits like from dependabot as signs of activity)
+* Code changes are secured by one of the following:
+  * [ ] **Only a small group of people have commit access:**
+  * [ ] **Code review is enforced:**
+* [ ] **No binary artifacts:** Binary artifacts are acceptable if they are so obviously not for use in production or not supposed to contain code that any production code referencing them would be obviously dangerous to a reviewer (e.g. image files or test assets).
```

Comment:
I can't really understand this sentence. Could you rephrase it?

(Should the first "so obviously" be just "obviously"? I also can't tell whether "that" refers to the "so... that" or to the "code" just before it.)

Reply:
done